src/status_im/contexts/wallet/account/tabs/view.cljs

```diff
@@ -11,6 +11,22 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn- on-collectible-press
+  [{:keys [id]}]
+  (rf/dispatch [:wallet/get-collectible-details id]))
+
+(defn- on-collectible-long-press
+  [{:keys [preview-url collectible-details]}]
+  (rf/dispatch [:show-bottom-sheet
+                {:content (fn []
+                            [options-drawer/view
+                             {:name  (:name collectible-details)
+                              :image (:uri preview-url)}])}]))
+
+(defn- on-end-reached
+  []
+  (rf/dispatch [:wallet/request-collectibles-for-current-viewing-account]))
+
```

Comment:
BTW, this was part of the solution to avoid the re-renders, previously they weren't stable references, so they caused a re-render on every collectible